src/navigation/MainNavigator.tsx

```diff
@@ -51,11 +52,19 @@ const settingsStackTransitionPreset = Platform.select({
   android: TransitionPresets.DefaultTransition,
 })
 
+const customPrefixes = env.DEEP_LINK_PREFIXES?.split(",") || []
+const allPrefixes = ["pathcheck://", "https://*.en.express/", ...customPrefixes]
```

Comment:
Per the notes from our contact at apple, it looks like the idea is to support universal links that integrate nicely with en express with this format: "https://appname.en.express".  Not 100% sure as we are not able to test locally without a real url to use, but we are hoping that 'react-navigation' will correctly handle this wildcard as that will save us the need to configure these hosts at build time.  Either way, I think it makes sense to take a pass at this in a future commit.